pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -154,7 +160,7 @@ func (vm *windows) FileExists(path string) (bool, error) {
 		return false, errors.Wrapf(err, "error while checking if the file %s exists on Windows VM %s. output: %s",
 			path, vm.ID(), out)
 	}
-	return (out == "True"), nil
+	return strings.TrimSpace(out) == "True", nil
```

Comment:
Please add this to the commit where you are introducing the helper and mention it in the commit message.